src/openms/source/ANALYSIS/OPENSWATH/TargetedSpectraExtractor.cpp

```diff
@@ -748,6 +756,12 @@ namespace OpenMS
 
     // select the best spectrum for each group of spectra having the same name
     selectSpectra(scored_spectra, ms2_features, extracted_spectra, extracted_features, compute_features);
+    
+    // NOTE/DEBUG: commenting out line 758 and uncommenting lines
```

Comment:
Not sure if I understand this but maybe enclosing this with an ifdef of a CLASSNAME_DEBUG macro would be better suited than a comment.

Reply:
I don't think it is needed.  I just added a small note for potential future work.